flixel/math/FlxPoint.hx

```diff
@@ -152,6 +152,19 @@ class FlxPoint implements IFlxPooled
 	}
 	
 	/**
+	 * Multiply this point by a scalar.
+	 * 
+	 * @param	scalar	How much times this point will be as it before.
+	 * @return	This point.
+	 */
+	public function multiply(scalar:Float):FlxPoint
```

Comment:
I'm aware, that doesn't make it any less redundant - both functions have the same implementation. Since `FlxVector` extends `FlxPoint`, `FlxVector` would have a `scale()` **and** a `multiply()`.

If anything, `scale()` should be moved to `FlxPoint()`.